config/services.php

```diff
@@ -33,7 +33,7 @@
     'google' => [
         'client_id' => '824567396641-9708860pecqvnhect7f9hknmdikcr94m.apps.googleusercontent.com',
         'client_secret' => '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',
-        'redirect' =>'http://localhost:8000/auth/google/callback',
+        'redirect' =>'http://127.0.0.1:8000/auth/google/callback',
```

Comment:
can you make it dynamic? like:
 url('/auth/google/callback')

Reply:
when I try to change dynamic, it will  show up like this 

Invalid Redirect: must end with a public top-level domain (such as .com or .org).
Invalid Redirect: must use a domain that is a valid [top private domain ](https://github.com/google/guava/wiki/InternetDomainNameExplained#public-suffixes-and-private-domains).